### PR TITLE
fix: Avoid deprecated properties in goreleaser integration config

### PR DESCRIPTION
**Pull request by reitzig**
`changelog.skip` has been deprecated in favor of `changelog.disable` since 2024-01-14

cf. https://goreleaser.com/deprecations/#changelogskip

**Check the following**
- [x] Keep 100% code coverage
- [x] Be properly formatted
- [x] Documentation changes are included
- [x] Include a change file if expected

**Additional context**
Any additional info that might help get your pull request merged.

### CODE REVIEW

**Comment by miniscruff** [comment]
Oh cool, can you also change the value in changie's config here https://github.com/miniscruff/changie/blob/main/.goreleaser.yml#L87

**Comment by reitzig** [comment]
ℹ️ This change hasn't been released yet; [goreleaser/goreleaser@v1.23.0](https://github.com/goreleaser/goreleaser/releases/tag/v1.23.0) is from last December.

Didn't notice that before, my bad. Suggest to hold back on merging until 1.24 drops.

**Comment by miniscruff** [comment]
Will do, I will mark this as draft and on hold until I notice the new release. Thanks for the help.

File: .changes/unreleased/Fixed-20240204-145414.yaml
Comment (miniscruff) [inline .changes/unreleased/Fixed-20240204-145414.yaml:1]: I have not been marking documentation updates in the changie changelog, so I will discard this file, thanks for considering it though.

Comment (reitzig) [reply .changes/unreleased/Fixed-20240204-145414.yaml:1]: The workflow complained about the missing file. Meant to ask but forgot ... so I guess you'd add the `skip changelog` label?

(Still figuring out how to set up my own repo. 😇)

Comment (miniscruff) [reply .changes/unreleased/Fixed-20240204-145414.yaml:1]: yea I added the label for you, I like making it an explicit allow sort of label rather then the opposite. You are safe to delete it now though.

**Comment by miniscruff** [comment]
For future miniscruff:

We are waiting on goreleaser 1.24 with the new config change, hopefully I get the notification for it.

File: .changes/unreleased/Fixed-20240204-145414.yaml
Comment (reitzig) [reply .changes/unreleased/Fixed-20240204-145414.yaml:1]: Gotcha, thanks!

**Comment by reitzig** [comment]
They done it! 🥳
➡️ [goreleaser/goreleaser@v1.24.0](https://github.com/goreleaser/goreleaser/releases/tag/v1.24.0)

**Comment by miniscruff** [comment]
Merged, will redeploy the website as well for that update.

**Comment by miniscruff** [comment]
Ok website is now also updated, thanks for the patch @reitzig

**Pull request merged**